### PR TITLE
[Mailer] Added OhMySMTP mailer details

### PR DESCRIPTION
**Pull request by paul-oms**
Refs https://github.com/symfony/symfony/pull/42710

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -112,11 +112,12 @@ Mailjet             ``composer require symfony/mailjet-mailer``
 Postmark            ``composer require symfony/postmark-mailer``
 SendGrid            ``composer require symfony/sendgrid-mailer``
 Sendinblue          ``composer require symfony/sendinblue-mailer``
+OhMySMTP            ``composer require symfony/ohmysmtp-mailer``
 ==================  ==============================================
 
-.. versionadded:: 5.2
```
Comment (OskarStark) [inline mailer.rst:117]: Please keep the 5.2 versionadded, copy it and create 5.4 below the 5.2, thanks

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: mailer.rst [hunk 2]
```diff
@@ -1326,6 +1329,10 @@ The following transports currently support tags and metadata:
 * Postmark
 * Sendinblue
 
+The following transports only support tags:
```
Comment (OskarStark) [inline mailer.rst:1336]: Only the following transports support tags:


?

Comment (paul-oms) [reply mailer.rst:1336]: OhMySMTP ignores Metadata headers, but you can add multiple Tags to an email https://docs.ohmysmtp.com/reference/send/#body-parameters https://docs.ohmysmtp.com/integrations/smtp/#tags

Comment (OskarStark) [reply mailer.rst:1336]: Sorry for being clear, I meant rewording the sentence by my proposal

Comment (paul-oms) [reply mailer.rst:1336]: Fixed :)

Comment (OskarStark) [reply mailer.rst:1336]: Ah while reading the paragraph before I realized your sentence was correct , sorry, can you please change it for a last time? Thanks

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
@paul-oms thanks for adding these contents ... and congrats on your first Symfony Docs contribution 🎉

**Comment by javiereguiluz** [comment]
I'm closing this manually because the tool we use didn't close it automatically, as usual.

I don't know the reason, because this looks like it was merged correctly in 5.4 (and 6.0) --> https://github.com/symfony/symfony-docs/commits/5.4

**Pull request closed without merging**